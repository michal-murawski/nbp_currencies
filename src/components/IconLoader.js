import React from 'react';
import { CircularProgress } from 'material-ui/Progress';

const IconLoader = () => <CircularProgress size={15} thickness={1} />;

export default IconLoader;
