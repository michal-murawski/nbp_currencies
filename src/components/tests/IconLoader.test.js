import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { IconLoaderRaw } from '../IconLoader';

const mockClasses = { loader: 'test_class' };

describe('components/IconLoader', () => {
  it('should render properly', () => {
    const wrapper = shallow(<IconLoaderRaw classes={mockClasses} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
