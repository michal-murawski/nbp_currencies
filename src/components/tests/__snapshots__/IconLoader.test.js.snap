// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/IconLoader should render properly 1`] = `
<WithStyles(CircularProgress)
  className="test_class"
  size={15}
  thickness={1}
/>
`;
